services/templeton/src/templeton/elasticsearch/elasticsearch.go

```diff
@@ -0,0 +1,86 @@
+package elasticsearch
+
+import (
+	"logcabin"
+
+	"encoding/json"
+	"github.com/mattbaird/elastigo/lib"
+
+	"templeton/database"
+	"templeton/model"
+)
+
+var (
+	logger = logcabin.New()
+)
+
+// Elasticer is a type used to interact with Elasticsearch
+type Elasticer struct {
+	es      *elastigo.Conn
+	baseURL string
+	index   string
+}
+
+// NewElasticer returns a pointer to an Elasticer instance that has already tested its connection
+// by making a WaitForStatus call to the configured Elasticsearch cluster
+func NewElasticer(elasticsearchBase string, elasticsearchIndex string) (*Elasticer, error) {
+	c := elastigo.NewConn()
+
+	err := c.SetFromUrl(elasticsearchBase)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
(Knowing absolutely nothing about Go, so I'm asking out of curiosity) I see these same 3 lines for checking errors sprinkled in multiple times across multiple files, is there an easier way to handle this?